cc-cli/ArgumentHandler.cs

```diff
@@ -219,7 +235,9 @@ public bool IsFilenameAKeyword(string filename)
             ["-u"] = "Units",
             ["--units"] = "Units",
             ["-t"] = "CcType",
-            ["--type"] = "CcType"
+            ["--type"] = "CcType",
```

Comment:
Is "CcType" customer facing? Wondering because the ReadMe just says "type." If we are changing it to "CcType", what is the reason for the change?

Reply:
This didn't change, what you are seeing is the addition of a comma.
We added a new option "-s", so a comma is needed to seperate them.
"CcType" is not user facing it is related to a property name.